tests/e2e/testfiles/scenarios.yaml

```diff
@@ -1875,6 +1875,47 @@
       run:
         - cert-csi test vio --sc op-e2e-pmax --chainLength 1 --chainNumber 1
 
+- scenario: "Install PowerMax Driver with Mount Credentials, Enable Authorization v1"
```

Comment:
Can auth v1 work with reverseproxy standalone deployment? Document (https://dell.github.io/csm-docs/docs/authorization/v1.x/configuration/powermax/ ) says `Update the DeployAsSidecar environment variable for the csipowermax-reverseproxy component to true.`